aw_analysis/query2_functions.py

```diff
@@ -214,6 +215,12 @@ def q2_sum_durations(events: list) -> timedelta:
     _verify_variable_is_type(events, list)
     return sum_durations(events)
 
+@q2_function
+def q2_sum_event_lists(events1: list, events2: list) -> List[Event]:
```

Comment:
`sum_event_list` is probably a bad name, this is rather appending two lists. It's also generic enough to not have to be a `event_list` so that shouldn't be a part of the name either.

Maybe name it `merge_lists` or `append_lists`?

Reply:
Or `concatenate`.

Reply:
It's not generic as it sorts by timestamp. If you give it an object without that property, it'll trigger an attribute error. That said, the name should probably include the fact that it returns a sorted list

Reply:
In that case I wonder why does it do two things in one function? There's a seperate function for sorting by timestamp.

Reply:
Seems I somehow missed the sort_by_timestamp. I'll separate it into two functions.